pynet/losses/generative.py

```diff
@@ -182,7 +183,8 @@ def compute_ll(self, p, data):
         data: torch.Tensor
             reference data.
         """
-        return - p.log_prob(data).sum(-1, keepdim=True)
+
+        return - p.log_prob(data).sum()
```

Comment:
Specify in the docstring that we compute the sum reduced log lokelihood
 return - BaseLoss._compute_ll(p, data).sum()